homeassistant/components/telegram_bot/__init__.py

```diff
@@ -277,12 +279,11 @@ def _render_template_attr(data, attribute):
         if msgtype == SERVICE_SEND_MESSAGE:
             yield from hass.async_add_job(
                 partial(notify_service.send_message, **kwargs))
-        elif msgtype == SERVICE_SEND_PHOTO:
+        elif msgtype == SERVICE_SEND_PHOTO or \
+             msgtype == SERVICE_SEND_VIDEO or \
```

Comment:
i am not sure what's the right thing to do here? can somebody please help?

Reply:
How about `msgtype in [SERVICE_SEND_PHOTO, SERVICE_SEND_VIDEO, SERVICE_SEND_DOCUMENT]:`

Reply:
actually, i got around it by putting the condition in brackets (that seems to be PEP-8 compliant). i think it's better than making a list this way. please let me know if you feel otherwise. :)

Reply:
That works too :+1: